FireboltNETSDK/FireboltClient1.cs

```diff
@@ -36,13 +36,13 @@ public class FireboltClient1 : FireboltClient
             "ENGINE_STATUS_PROVISIONING_STARTED", "ENGINE_STATUS_PROVISIONING_PENDING",
             "ENGINE_STATUS_PROVISIONING_FINISHED", "ENGINE_STATUS_RUNNING_REVISION_STARTING"
     };
-    public FireboltClient1(FireboltConnection connection, string id, string secret, string endpoint, string? env, string account, HttpMessageInvoker httpClient) : base(connection, id, secret, endpoint, env, null, httpClient)
+    public FireboltClient1(FireboltConnection connection, string id, string secret, string endpoint, string? env, string account, HttpClient httpClient) : base(connection, id, secret, endpoint, env, null, httpClient)
     {
         _settings = new Lazy<JsonSerializerSettings>(new JsonSerializerSettings());
         _account = account;
     }
 
-    public override async Task<ConnectionResponse> ConnectAsync(string? engineName, string databaseName, CancellationToken cancellationToken)
+    public override async Task<ConnectionResponse> ConnectAsync(string? engineName, string database, CancellationToken cancellationToken)
```

Comment:
Why are we renaming this? Looks consistent with `engineName` for me.

Reply:
I would avoid touching 1.0 unless we need to. fingers crossed it will be deprecated soon enough.